mitosheet/src/components/elements/MultiToggleBox.tsx

```diff
@@ -242,6 +248,7 @@ const MultiToggleBox = (props: {
                     loading={props.loading}
                     isSubset={props.isSubset}
                     message={props.message}
+                    _emptyMessage={props.emptyMessage}
```

Comment:
Can we change the name of this param to emptyMessage? And then change the variable name used in the function?